easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.10.3-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '1.10.3'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+#sources = [SOURCELOWER_TAR_GZ]
+sources = ['openmpi-%(version)src4.tar.gz']
```

Comment:
this obviously needs to be changed before this can be merged...

final release of OpenMPI 1.10.3 is planned for June 10th (cfr. https://www.open-mpi.org/community/lists/devel/2016/06/19068.php)